great-idea-website/index.html

```diff
@@ -1,63 +1,117 @@
-
-
-  Product
-  Aliquam elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi. Suspendisse eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.
-
-  Vision
-  Aliquam elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi. Suspendisse eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.
-
-  Contact
-  123 Way 456 Street
-  Somewhere, USA
-  1 (888) 888-8888
-  sales@greatidea.io
-
-  Copyright Great Idea! 2018
-  
-  </body>
-</html>
\ No newline at end of file
+    <head>
+        <meta charset="utf-8" />
+
+        <title>Great Idea!</title>
+
+        <link
+            href="https://fonts.googleapis.com/css?family=Bangers|Titillium+Web"
+            rel="stylesheet"
+        />
+        <link rel="stylesheet" href="css/index.css" />
+
+        <!--[if lt IE 9]>
+            <script src="https://cdnjs.cloudflare.com/ajax/libs/html5shiv/3.7.3/html5shiv.js"></script>
+        <![endif]-->
+    </head>
+
+    <body>
+        <div class="container">
+            <nav>
+                <a href="">Services</a> <a href="">Products</a>
+                <a href="">Vision</a> <a href="">Features</a>
+                <a href="">About</a> <a href="">Contact</a>
+                <img
+                    class="logo"
+                    src="img/logo.png"
+                    alt="Great Idea! Company logo."
+                />
+            </nav>
+            <header>
+                <div class="big-text">
+                    <h1>Innovation On Demand</h1>
+                    <button>Get Started</button>
+                </div>
+                <img
+                    class="header-img"
+                    src="img/header-img.png"
+                    alt="code snippit."
+                />
+            </header>
+            <hr />
+
+            <section class="top-section">
+                <div class="left-div">
+                    <h3>Features</h3>
+                    <p>
+                        Aliquam elementum magna eros, ac posuere elvit tempus
+                        et. Suspendisse vel tempus odio, in interdutm nisi.
+                        Suspendisse eu ornare nisl. Nullam convallis augue
+                        justo, at imperdiet metus scelerisque quis.
+                    </p>
+                </div>
+
+                <div class='right-div'>
+                    <h3>About</h3>
+                    <p>
+                        Aliquam elementum magna eros, ac posuere elvit tempus
+                        et. Suspendisse vel tempus odio, in interdutm nisi.
+                        Suspendisse eu ornare nisl. Nullam convallis augue
+                        justo, at imperdiet metus scelerisque quis.
+                    </p>
+                </div>
+            </section>
+
+            <img
```

Comment:
I like how you're formatting these image tags. I know you didn't write this alt, but fun fact: for image alts, you don't have to say "Image of"; screen readers will prepend the alt with something along those lines.